mlir/lib/Conversion/TosaToRock/TosaToRock.cpp

```diff
@@ -215,6 +215,58 @@ makeRockConv(ConversionPatternRewriter &rw, Operation *op, Value input,
   return cop;
 }
 
+static bool isTosaReduction(Operation *op) {
+  return isa<tosa::ReduceMaxOp, tosa::ReduceSumOp, tosa::ReduceMinOp,
+             tosa::ReduceProdOp, tosa::ReduceAllOp, tosa::ReduceAnyOp>(op);
+}
+
+static Value traceToRes(Value tensor, DenseMap<Value, Value> &cache,
```

Comment:
nit : what does Res means in the context ? 
I thought it was result but you are tracing to arguments right ?

Reply:
In TosaToRock we trace to results (the returnop operands). In GemmToGridwise we trace to arguments.

Reply:
i see. alright.